Version 0.1a

Tanais opened the pull request.

as-tanais committed:
FirstVers

as-tanais committed:
Make HW.

as-tanais committed:
+Личные сообщения
+Списко клиентов. (Все из урока 8)
-Пока не получилось сделать авто кик не авторизированых пользователей.

efserg commented on src/main/java/ru/as/homeworks/lan_chat_start/server/ClientHandler.java:
```diff
@@ -0,0 +1,128 @@
+            this.chatServer = chatServer;
+            in = new DataInputStream(socket.getInputStream());
+            out = new DataOutputStream(socket.getOutputStream());
+
```
Как только клиент подключился (но еще не авторизовался), мы попадаем в этот конструктор. Как раз здесь логично сделать новый поток, в котором и заснуть на требуемое время. Когда поток проснется, он первым делом проверит, авторизовался ли пользователь (как реализовать эту проверку - придумайте), и там уже решить, что делать

efserg reviewed (changes requested):
Александр, жду реализацию таймера, пока оценку не ставлю

as-tanais committed:
+ Поток на Таймер. Пока с ошибкой. Нужно еще время на подумать.

as-tanais committed:
+ Поток на Таймер. Пока с ошибкой. Нужно еще время на подумать.
+ Не понимаю как завершить потом и разрыв соеденения коректно. Клиента кикакет но программа плодит ошибку.

as-tanais committed:
+ Не понимаю как завершить потом и разрыв соеденения коректно. Клиента кикакет но программа плодит ошибку.

+ Запись истроии в файл.

as-tanais committed:
+ Не понимаю как завершить потом и разрыв соеденения коректно. Клиента кикакет но программа плодит ошибку.

+ Запись истроии в файл.
